app/install/microsoft-azure.md

```diff
@@ -0,0 +1,99 @@
+                                                  KONG_PG_PASSWORD="<password>" \
+                          --command-line "kong migrations up"
+    ```
+    In this example, we are using a PostgreSQL database running on [Azure Database for PostgreSQL](#running-postgresql-on-azure-with-azure-database-for-postgresql).
+
+1. **Start Kong**
+
+    ```bash
+    $ az container create --name kong-gateway /
+                          --dns-name-label kong-gateway /
+                          --resource-group kong-sandbox /
+                          --image kong:latest /
+                          --port 8000 8443 8001 8444 /
+                          --environment-variables KONG_PG_HOST="<instance-name>.postgres.database.azure.com" /
+                                                  KONG_PG_USER="<username>" /
+                                                  KONG_PG_PASSWORD="<password>" /
+                                                  KONG_PROXY_ACCESS_LOG="/dev/stdout" /
+                                                  KONG_ADMIN_ACCESS_LOG="/dev/stdout" /
+                                                  KONG_PROXY_ERROR_LOG="/dev/stderr" /
+                                                  KONG_ADMIN_ERROR_LOG="/dev/stderr" /
+                                                  KONG_ADMIN_LISTEN="0.0.0.0:8001, 0.0.0.0:8444 ssl"
+    ```
+
+    The `8000`, `8001`, `8443`, and `8444` will be forwarded to the container.
+
+    <div class="alert alert-warning">
+      <div class="text-center">
+        <strong>Note</strong>: This will expose both the proxy and the Admin API on the default ports. This can have security implications.
+      </div>
+    </div>
+
+1. **Use Kong**
+
+    That's it - You can now use Kong by browsing to `<dns-label>.westeurope.azurecontainer.io`.
+
+    Quickly learn how to use Kong with the [5-minute Quickstart](/latest/getting-started/quickstart).
+
+# Running PostgreSQL on Azure with Azure Database for PostgreSQL
+Azure Database for PostgreSQL is a great way to use a managed PostgreSQL in the Azure Cloud.
+
+Here are the simple steps to provision one:
+
+1. Go to the <a href="https://portal.azure.com" target="blank">Azure Portal</a>
+1. Create a new "Azure Database for PostgreSQL" instance
+1. Go to "Connection Security" and enable access to Azure services
+1. Create a new database called "kong" by using your favorite tool
+
+<div class="alert alert-warning">
+  <div class="text-center">
+    <strong>Note</strong>: Before connecting to your new database, make sure your IP address is whitelisted in "Connection Security"
+  </div>
+</div>
+
+# Running Cassandra on Azure with Azure Cosmos Db
+Currently, Azure Cosmos Db is not supported as a Cassandra data store.
+
+<div class="alert alert-info">
+  <div class="text-center">
+    <strong>Note</strong>: See <a href="https://github.com/Kong/docker-kong/issues/188" target="blank">#188</a> for more information.
+  </div>
+</div>
```

Comment:
Needs a blank line at the end.

Reply:
I added a blank line but seems like the diff does not show it at the moment.